src/app/views/authentication/profile/Profile.tsx

```diff
@@ -103,11 +104,17 @@ export class Profile extends Component<IProfileProps, IProfileState> {
       graphExplorerMode,
     }: any = this.props;
 
+    const delegatedPermissions = this.props.permissionModeType === DISPLAY_DELEGATED_PERMISSIONS;
```

Comment:
better variable name: `isDelegatedPermissions`

Reply:
actually couldn't we simply make `this.props.permissionModeType` as an Enum?

Reply:
There's only two permissions mode types, delegated and application, so I went with Boolean to represent them. But I will change the var name to be more descriptive!